internal/utils/validation/film.go

```diff
@@ -16,3 +18,25 @@ func ValidateRating(fl validator.FieldLevel) bool {
 	}
 	return false
 }
+
+func ValidateSpecialFeatures(fl validator.FieldLevel) bool {
+    validFeatures := map[string]struct{}{
+        "Trailers":       {},
+        "Commentaries":   {},
+        "Deleted Scenes": {},
+        "Behind the Scenes": {},
+    }
+
+    features := fl.Field().String()
+    featureList := strings.Split(features, ",")
+
+    for _, feature := range featureList {
+        feature = strings.TrimSpace(feature)
```

Comment:
- already remove trim space
- i use string here because i want to split each special features into individuals strings, so i can check whether they are the right value or not